py_ocpi/commands/v_2_2_1/api/cpo.py

```diff
@@ -1,10 +1,32 @@
-from fastapi import APIRouter
+from typing import Union
+
+from fastapi import APIRouter, Depends
+from pydantic import ValidationError
+
+from py_ocpi.core.dependencies import get_crud, get_adapter
+from py_ocpi.core.enums import ModuleID
+from py_ocpi.core.schemas import OCPIResponse
+from py_ocpi.core import status
+from py_ocpi.commands.v_2_2_1.enums import CommandType
+from py_ocpi.commands.v_2_2_1.schemas import CancelReservation, ReserveNow, StartSession, StopSession, UnlockConnector
 
 router = APIRouter(
     prefix='/commands',
 )
 
 
-@router.get("/")
-def get_commands():
-    return "commands app created!"
+@router.post("/{command}", response_model=OCPIResponse)
+async def receive_command(command: CommandType,
+                          data: Union[CancelReservation, ReserveNow, StartSession, StopSession, UnlockConnector],
+                          crud=Depends(get_crud), adapter=Depends(get_adapter)):
+    try:
+        response = await crud.create(ModuleID.commands, command, data.dict())
```

Comment:
create method is the following:
```
@classmethod
async def create(cls, module: ModuleID, data, *args, **kwargs) -> Any:
    ...
```
so the second parameter must be data. and we need to pass command like `command=command` to have it in kwargs